src/serac/physics/utilities/finite_element_state.hpp

```diff
@@ -105,6 +105,14 @@ class FiniteElementState {
    */
   FiniteElementState(mfem::ParMesh& mesh, mfem::ParGridFunction& gf, const std::string& name = "");
 
+  /**
+   * @brief Minimal constructor for a FiniteElementState given an already-existing state
+   * @param[in] mesh The problem mesh (object does not take ownership)
+   * @param[in] fe_state The state for the new state to copy
```

Comment:
You have some mismatching parameter names here:

* doxygen comment: fe_state
* declaration: gf
* implementation: original_state